lib/DockerWrapper.js

```diff
@@ -307,4 +310,34 @@ module.exports = {
       });
     };
   },
+
+  // This function takes an optional docker manager argument.
+  // We do this so that our websockets can obtain a manager instance
+  // on their own when they first connect to a client, that way
+  // we hit the database for the manager (and app) only once instead
+  // of every second when we call this method.
+  appIsHealthy: async(app, docker) => {
+    if (docker === undefined) {
+      docker = await getManagerNodeInstance();
+    }
+
+    const services = await docker.listServices();
+    const service = services.filter(service => {
+      return service.Spec.Name === app.title + '_web';
+    })[0];
```

Comment:
Minor: Could just use `.find` here so you don't need to do the `[0]` bit.

Reply:
Good point, I always forget about `find`